### PR TITLE
Use correct CircleCI environment variable for pull request detection

### PR DESCRIPTION
**Pull request by esetnik**
https://circleci.com/docs/variables/#built-in-environment-variables:~:text=CIRCLE_PULL_REQUESTS

### CODE REVIEW

**Comment by esetnik** [comment]
not stale

**Comment by ninadbstack** [comment]
@esetnik can you raise a support ticket as well for this ? In order to get this merged we will need to once go through the circle ci docs to see what all has changed and if there is any on self hosted older versions which will continue to need older env vars.

**Comment by esetnik** [comment]
> @esetnik can you raise a support ticket as well for this ? In order to get this merged we will need to once go through the circle ci docs to see what all has changed and if there are any incompatibility issues on on self hosted older versions which will continue to need older env vars.
> 
> If everything looks good then we will get this merged!

I opened a support ticket and haven't heard back.

**Review by pankaj443 — APPROVED** [review]
Checked docs and confirmed  updated env vars are valid

**Review by ninadbstack — APPROVED** [review]

**Comment by ninadbstack** [comment]
Added a newline [ basically blank commit ] to allow triggering workflows

File: packages/env/src/environment.js [hunk 1]
```diff
@@ -212,15 +212,13 @@ export class PercyEnv {
...
     let total = parseInt(this.vars.PERCY_PARALLEL_TOTAL, 10);
-    if (!Number.isInteger(total)) total = null;
 
-    if (total === null) {
-      switch (this.ci) {
-        case 'circle':
-          total = this.vars.CIRCLE_NODE_TOTAL;
-      }
+    if (this.ci === 'circle') {
+      total = parseInt(this.vars.CIRCLE_NODE_TOTAL, 10);
```
[the remark this answers is not in the document]
Comment (pankaj443) [reply packages/env/src/environment.js:217]: Have updated in seperate PR

**Comment by pankaj443** [comment]
Hey @esetnik we have released v1.30.5-beta.1 with above changes.

**Pull request merged**